_episodes/05-writing-scripts.md

```diff
@@ -60,10 +60,10 @@ The text at the bottom of the screen shows the keyboard shortcuts for performing
 
 > ## Which Editor?
 >
-> When we say, "`nano` is a text editor," we really do mean "text": it can
+> When we say, "`nano` is a text editor," we really do mean "text": `nano` can
```

Comment:
Maybe `it` is more appropriate here, since the sentence already establishes `nano` as the subject.